test/models/pawn_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class PawnTest < ActiveSupport::TestCase
+  test " white_pawn_legal_move" do
+    pawn = FactoryGirl.create(:pawn, :color => "White", :row_position => 1, :col_position => 0)
+    assert pawn.legal_move?(2, 0)
+    # test backward movement
+    refute pawn.legal_move?(0, 0)
+    # test horizontal movement
+    refute pawn.legal_move?(1, 1)
+    # test diagonal movement
+    refute pawn.legal_move?(2, 1)
+    # test two square movement
+    refute pawn.legal_move?(3, 0)
```

Comment:
We will eventually want to assert the test on line 14.  Might be good to add a TODO in the comment on line 13.